### DISCUSSION TITLE
Wrong detection of digital an analog values

### DISCUSSION THREAD

**Discussion by docolli** [jomjol/AI-on-the-edge-device#1832] [Q&A]
Running on v13.0.8 the recognition was fine for several weeks until the last days. Suddenly the recognition is completely wrong. See picture:

![watermeter_2](https://user-images.githubusercontent.com/78646938/212465707-3cadab2f-41d2-4eca-9ca3-770c48bd12cf.png)

![watermeter_1](https://user-images.githubusercontent.com/78646938/212465622-b41511d9-a6d7-463e-af82-30003c037aa2.png)

What can I do? Is there a simple way to reset the AI recognition, which seems to be skrewed up? Maybe delete some files on the SD card and restart ESP?

**Comment by docolli** [comment]
I changed the AI modell in config and reboot ESP. Now detection is fine again. Old and new AI model values:
![watermeter_3](https://user-images.githubusercontent.com/78646938/212466328-c04208e8-4b5b-45e1-8179-218b3570030f.png)
![watermeter_4](https://user-images.githubusercontent.com/78646938/212466332-c60a7fb2-9fe6-48fe-a583-40b96590b0c6.png)

Can someone explain which model is best for my meter?

**Comment by ghost** [comment]
Look at https://jomjol.github.io/AI-on-the-edge-device-docs/Choosing-the-Model/ and https://jomjol.github.io/AI-on-the-edge-device-docs/Neural-Network-Types/
